tools/data-migration-deployment.md

```diff
@@ -0,0 +1,324 @@
+    The key fingerprint is:
+    SHA256:eIBykszR1KyECA/h0d7PRKz4fhAeli7IrVphhte7/So tidb@172.16.10.49
+    The key's randomart image is:
+    +---[RSA 2048]----+
+    |=+o+.o.          |
+    |o=o+o.oo         |
+    | .O.=.=          |
+    | . B.B +         |
+    |o B * B S        |
+    | * + * +         |
+    |  o + .          |
+    | o  E+ .         |
+    |o   ..+o.        |
+    +----[SHA256]-----+
+    ```
+
+## Step 3: Download DM-Ansible to the Control Machine
+
+1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory.
+
+2. Run the following command to download DM-Ansible.
+
+    ```bash
+    $ wget http://download.pingcap.org/dm-ansible.tar.gz
+    ```
+
+## Step 4: Install Ansible and its dependencies on the Control Machine
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+It is required to use `pip` to install Ansible and its dependencies, otherwise a compatibility issue occurs. Currently, DM-Ansible is compatible with Ansible 2.5 or later.
+
+1. Install Ansible and the dependencies on the Control Machine:
+
+    ```bash
+    $ tar -xzvf dm-ansible.tar.gz
+    $ cd /home/tidb/dm-ansible
+    $ sudo pip install -r ./requirements.txt
+    ```
+
+    Ansible and the related dependencies are in the `dm-ansible/requirements.txt` file.
+
+2. View the version of Ansible:
+
+    ```bash
+    $ ansible --version
+    ansible 2.5.0
+    ```
+
+## Step 5: Configure the SSH mutual trust and sudo rules on the Control Machine
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+1. Add the IPs of your deployment target machines to the `[servers]` section of the `hosts.ini` file.
+
+    ```
+    $ cd /home/tidb/dm-ansible
+    $ vi hosts.ini
+    [servers]
+    172.16.10.71
+    172.16.10.72
+    172.16.10.73
+
+    [all:vars]
+    username = tidb
+    ```
+
+2. Run the following command and input the `root` user account password of your deployment target machines.
+
+    ```bash
+    $ ansible-playbook -i hosts.ini create_users.yml -u root -k
+    ```
+
+    This step creates the `tidb` user account on the deployment target machines, configures the sudo rules and the SSH mutual trust between the Control Machine and the deployment target machines.
+
+## Step 6: Download DM and the monitoring component installation package to the Control Machine
+
+Make sure the Control Machine is connected to the Internet and run the following command:
+
+```bash
+ansible-playbook local_prepare.yml
+```
+
+## Step 7: Edit the `inventory.ini` file to orchestrate the DM cluster
+
+Log in to the Control Machine using the `tidb` user account, and edit the `/home/tidb/dm-ansible/inventory.ini` file to orchestrate the DM cluster.
+
+> **Note:** It is required to use the internal IP address to deploy. If the SSH port of the target machines is not the default 22 port, you need to add the `ansible_port` variable. For example,
+>
+> ```ini
```

Comment:
```suggestion
>     ```ini
```

Reply:
@CaitinChen No indentation issue here. So I'll keep the original version. :)